core/src/main/java/bisq/core/app/BisqSetup.java

```diff
@@ -416,7 +416,9 @@ private void initWallet() {
                 requestWalletPasswordHandler.accept(aesKey -> {
                     walletsManager.setAesKey(aesKey);
                     walletsSetup.getWalletConfig().maybeAddSegwitKeychain(walletsSetup.getWalletConfig().btcWallet(),
-                            aesKey);
+                            aesKey, false);
+                    walletsSetup.getWalletConfig().maybeAddSegwitKeychain(walletsSetup.getWalletConfig().bsqWallet(),
```

Comment:
Maybe we can handle that in walletsManager, which is used for operations which should be applied to both wallets, so it is closer to the wallet domain.

Reply:
Yes, it would probably be neater to put it there. I'll push a commit for that shortly.